app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
Nice job using AR. We will want to move this to our `ingredients` model as its own method. So something like
```ruby
def self.sort_alphabetically
  order(:name)
end
```